Remove references to SCA Part 2 (#2909)
theforeman/foreman-documentation: guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc

[id="Creating_an_Activation_Key_to_Consume_the_Content_View_{context}"]
= Creating an activation key to consume the content view

> Create an activation key to assign various attributes to hosts during registration.

Reviewer: This procedure seems to be specific to this workflow, and not general. I think we could actually just leave this sentence out.

Below, need to remove

```
. Click the *Subscriptions* tab on the Activation Key's details page.
. Click the *Add* tab.
. Select your new *CentOS Stream* product in the table.
. Click *Add Selected* to save.
```

Author: @jeremylenz Do you have any suggestions for a new opening sentence to replace `Create an activation key to assign various attributes to hosts during registration.` ?

Reviewer: The procedure is guiding you through creating an AK so that your host consumes the content view, so I couldn't really think of anything to replace it that didn't just re-state the title. Do we have to have an opening sentence?

This entire chapter ("Basic Content Management Workflow") seems designed so that you follow all the procedures in order, rather than just picking out one. 

I guess we could say something like

```
Create another activation key to assign hosts to the content view you created in the previous procedure.
```

Author: Yeah we need an intro of some kind. I think what you said here works perfectly.

> Create an activation key to assign various attributes to content managed by {Project}.

Reviewer: ```suggestion
Create an activation key to assign various attributes to hosts at registration time.
```

The attributes are assigned to hosts, not to content.

> Create another activation key to assign hosts to the content view you created in {ContentManagementDocURL}Creating_a_Content_View__content-management[Creating a content view] in _{ContentManagementDocTitle}_. 

Reviewer 2: ```suggestion
Create another activation key to assign hosts to the content view you created in {ContentManagementDocURL}Creating_a_Content_View_content-management[Creating a content view] in _{ContentManagementDocTitle}_. 
```
This _might_ even fix the linkchecker failure.

Reviewer 2: Wait. But shouldn't this be an xref? This module is in _Managing content_, isn't it? (It's hard to check without a preview.)

Author: You are correct. I was going back and forth between Hosts and Content that I must have mixed them up.
